docs/reference/setup/install/zip-targz.asciidoc

```diff
@@ -57,6 +57,9 @@ and can be stopped by pressing `Ctrl-C`.
 
 include::check-running.asciidoc[]
 
+Log printing to `STDOUT` can be disabled using the `-q` or `--quiet`
```

Comment:
Nit: can we either make this `stdout` so it's not shouting, or just say "standard output"?